codegen.yml

```diff
@@ -0,0 +1,23 @@
+overwrite: true
+schema: "http://localhost:8080/graphql"
+documents: './src/app/graphql/queries.ts'
+generates:
+  ./src/app/graphql/types.tsx:
+    plugins:
+      - add:
+          content:
+          - ''
+          - 'export namespace GraphQL {'
+      - add:
+          placement: 'append'
+          content: '}'
+      - "typescript"
+      - "typescript-operations"
+      - "typescript-react-apollo"
+    config:
+      withHOC: false
+      withHooks: true
+      withComponent: false
+  ./src/app/graphql/graphql.schema.json:
```

Comment:
```suggestion
 
```

We don't need to generate `graphql.schema.json` file in ui. We need only generate types.